### ISSUE TITLE
Simplify the concept of "group of people" across Nextcloud

### ISSUE THREAD

**Issue by marcoambrosini** [nextcloud/server#49488]
## The problem 
**Currently we have two definitions of "group of people" in Nextcloud: Teams and Groups. This creates confusion among users.**

## Possible solutions
* Call all groups of people "Teams". Does the non admin user need to know whether they're sharing to an admin-managed group or to a team-member-managed one?
    - To help differentiate, work with team premissions or team "types" using adjectives. 
* One unified interface to create and edit teams. Only admins will retain the ability to create 
    - Offer presets with different permissions set at creation, e.g. Microsoft teams:
    - ![Image](https://github.com/user-attachments/assets/23281806-4ba2-4d83-8d26-defbf1938fa1)
 
* Rename "Groupfolders" into "team folders"

**Comment by Ornanovitch** [comment]
I completely agree with the thoughts presented here: groups and teams tend to have parity of functionality from the user's point of view, and as such there should be the same word for both. The difference would be that only admin-created teams (today's groups) would be used to delegate admin powers and to prohibit or authorize applications. The Nextcloud ecosystem would gain enormously in legibility and simplicity.

On the other hand, I don't understand why you start renaming applications as complex as Group folders before you've clarified the core of the problem. I'm afraid this will add to the confusion: non-admin users currently using teams on an instance where Groups folders is installed will easily believe that it's possible to create team folders like those created by the admin, which doesn't look like it's going to happen any time soon (and it would have to be possible before renaming the application). There isn't even a team resource provider for Group folders yet, so all this seems a little rushed...

Groups and teams aren't the same thing yet, so why start renaming now? Maybe I've missed something - don't hesitate to let me know! Anyway, I hope there'll be a real big decision on all this soon, Nextcloud deserves it :)

Translated with DeepL.com (free version)

**Comment by Ornanovitch** [comment]
> One unified interface to create and edit teams. Only admins will retain the ability to create

AFAK Teams has been created precisely in order to give to users the possibility to create "groups", why such a step back?

**Comment by marcoambrosini** [comment]
Hi @Ornanovitch you're right, it's going to create some confusion but it's a small step in the direction of unification of this concept into a single term. Things are pretty confusing as they are now too. Starting right away with the renaming is also going to create a sense of urgency for the rest of components to catch up as quickly as possible. 

> non-admin users currently using teams on an instance where Groups folders is installed will easily believe that it's possible to create team folders like those created by the admin

Maybe we can add a warning on info box somewhere in the interface informing users that they need to request creation of a group folder to an admin.

**Comment by joshtrichards** [comment]
Related: 

- #13478

**Comment by ornanovitch** [comment]
Related: https://help.nextcloud.com/t/understanding-contacts-groups-teams-circles-nc32/242195

I'm afraid this topic is going to become more and more a big UX issue.
